projects/list-builder-view-checklist/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@skyux/list-builder-view-checklist",
+  "version": "5.0.0-beta.0",
+  "peerDependencies": {
+    "@angular/common": "^12.2.0",
+    "@angular/core": "^12.2.0",
+    "@angular/forms": "^12.2.0",
+    "@skyux/forms": "^5.0.0-beta.0",
```

Comment:
These versions should match what's in the workspace's dependency list.

Reply:
I've not been doing that for all repos... is that a problem?

Reply:
Done.

Reply:
Yes, because it means we test with one version but require a lower one, meaning we could be using a new feature in our tests that doesn't exist in the peer dependency we say we support. In this case, we're citing versions of SKY UX libraries that haven't been ported to partial Ivy which we know will cause problems for consumers.